src/workspace.js

```diff
@@ -422,6 +423,17 @@ module.exports = class Workspace extends Model {
       this.didChangeActivePaneItem(item)
       this.emitter.emit('did-change-active-pane-item', item)
     }
+
+    if (paneContainer === this.getCenter()) {
+      const activeTextEditorChanged =
+        (item instanceof TextEditor) || (this.activeTextEditor instanceof TextEditor)
```

Comment:
Thanks, @maxbrunsfeld. :bow:

I pushed up a few updates that attempt to incorporate these suggestions. If these updates are different from what you had in mind, just let me know.